lib/scryptenc/scryptenc_cpuperf.c

```diff
@@ -34,6 +34,7 @@
 #include "crypto_scrypt.h"
 #include "monoclock.h"
 
+#include "scryptenc.h"
```

Comment:
I considered moving all the `#define` lines into a new `scryptenc_errno.h` to avoid having to add this line, but I couldn't think of a sensible case in which somebody  would want to include `scryptenc.h` but not have access to the error definitions.

Reply:
The normal reason for putting them into `scryptenc_errno.h` would be the opposite -- to allow `scryptenc_cpuperf.c` to get the error definitions without being polluted with the rest of `scryptenc.h`.

But in this case I don't think that the pollution-avoidance benefit would justify having another file, so this is fine.